server/utils/placementRequests/utils.test.ts

```diff
@@ -28,7 +28,7 @@ describe('utils', () => {
       searchButton(placementRequest)
 
       expect(utils.linkTo).toHaveBeenCalledWith(
-        paths.placementRequests.beds.search,
+        paths.v2Match.placementRequests.spaces.search,
```

Comment:
I think we could make a good case for saying that the resource we're creating here is a "space search", or "space-search". i.e. search is the noun, rather than the verb. We're making a search and the specific type of search is a space-search...